docusaurus/docs/welcome/instructions/android-app-signing.md

```diff
@@ -0,0 +1,68 @@
+---
+id: android-app-signing
+title: Android app signing
+sidebar_label: Android app signing
+---
+
+To distribute an Android app via Google Play store it is required to digitally sign it with a certificate. If you're not yet familiar with how app signing works, have a look at [Android Developers' official documentation](https://developer.android.com/studio/publish/app-signing). 
+
+> ℹ️ Debug builds are automatically signed but your app should have its own, unique secure signing configuration for release builds. Otherwise, you won't be able to publish your app to Google Play. 
+
+The first step is to [generate an upload key and a keystore](https://developer.android.com/studio/publish/app-signing#sign-apk) and follow the instructions and recommendations therein. The second step is to configure the boilerplate to use them.
+
+> ❗️ Keep your keystore file private and in safe place! Never check it in into source control. By default, files ending in `.jks` and `.keystore` are already added to `.gitignore`.
+
+> 💡 You can securely save your keystore file, keystore password and upload key password in a password manager such as 1Password.
+
+The boilerplate offers three configuration options to sign release builds:
+
+1. Environment variables
+2. Properties file
+3. Properties file and macOS keychain
```

Comment:
If we did for Windows it would make sense to do it for Linux as well. I'd rather not spend time on that for the time being as it's not a priority. While the keychain enhances security overall, I think having something in place for macOS for now is something we can live with. What we can do, however, is indicate strategy 3 as the recommended approach to store passwords. Also, when we have a CI/CD pipeline up and running, the app will be signing on CD.